GitExtUtils/ArgumentString.cs

```diff
@@ -15,9 +15,9 @@ private ArgumentString(string arguments)
             Arguments = arguments ?? throw new ArgumentNullException(nameof(arguments));
         }
 
-        public static implicit operator ArgumentString(string args) => new ArgumentString(args);
+        public static implicit operator ArgumentString(string? args) => new ArgumentString(args ?? "");
         public static implicit operator ArgumentString(ArgumentBuilder args) => new ArgumentString(args.ToString());
-        public static implicit operator string?(ArgumentString args) => args.Arguments;
+        public static implicit operator string(ArgumentString args) => args.Arguments ?? "";
         public override string ToString() => Arguments ?? "";
```

Comment:
Remove null values from implicit conversions. The previous annotations introduced errors in consuming code that looked weird. It appeared as though a struct (`ArgumentString`) was somehow nullable.